### PR TITLE
Add cloudrobotics-core for bazel compatibility test runs

### PR DESCRIPTION
**Pull request by ensonic**

### CODE REVIEW

**Comment by ensonic** [comment]
Let me know if the other fields need to be set and what would be a good template to get started.

File: buildkite/bazelci.py [hunk 1]
```diff
@@ -93,6 +93,10 @@
         "http_config": "https://raw.githubusercontent.com/bazelbuild/continuous-integration/master/buildkite/pipelines/clion-postsubmit.yml",
         "pipeline_slug": "clion-plugin",
     },
+    "cloud-robotics core": {
+        "git_repository": "https://github.com/googlecloudrobotics/core.git",
```
[the remark this answers is not in the document]
Comment (ensonic) [reply buildkite/bazelci.py]: Added

**Comment by ensonic** [comment]
Thanks for the help.

**Comment by philwo** [comment]
@ensonic Can you check the log of the first run: https://buildkite.com/bazel/cloud-robotics-core/builds/1#09d8898f-e928-4282-92f5-0292c5e46d5e ?

It seems like we have to generate a "config.bzl" before we can build the project - how does that work? 

If necessary, we can run arbitrary shell commands via the `shell_commands` attribute in the https://github.com/bazelbuild/continuous-integration/blob/master/buildkite/pipelines/cloud-robotics-postsubmit.yml config. :)

**Comment by ensonic** [comment]
I'll send another PR. with this. The reason for this is https://github.com/bazelbuild/rules_docker/issues/683
For the test we can put anything there.

**Comment by laurentlb** [comment]
Can we make sure the projects work with incompatible flags, before we add them to the CI?

https://github.com/googlecloudrobotics/core/issues/2

**Pull request merged**